### PR TITLE
check for consumption API existence

### PR DESCRIPTION
**Pull request by phiSgr**

### CODE REVIEW

**Review by callicles — COMMENTED** [review]
As talked in the review - let's change this to have the feedback in the dev logs - not in HTTP response

File: apps/framework-cli/src/cli/local_webserver.rs [hunk 1]
```diff
@@ -337,13 +363,16 @@ async fn ingest_route(
     }
 }
 
+#[allow(clippy::too_many_arguments)]
```
Comment (phiSgr) [inline apps/framework-cli/src/cli/local_webserver.rs]: I still think making the `project` a global (immutable) variable is a good idea.

File: apps/framework-cli/src/cli/local_webserver.rs [hunk 2]
```diff
@@ -337,13 +363,16 @@ async fn ingest_route(
...
 async fn router(
     req: Request<hyper::body::Incoming>,
     current_version: String,
     route_table: &RwLock<HashMap<PathBuf, RouteMeta>>,
+    consumption_apis: &RwLock<HashSet<String>>,
     configured_producer: ConfiguredProducer,
     console_config: ConsoleConfig,
```
Comment (callicles) [inline apps/framework-cli/src/cli/local_webserver.rs]: Did you rebase? The console config should be gone and that might enable you to not have to add the clippy argument

File: apps/framework-cli/src/cli/local_webserver.rs (hunk 1 above)
Comment (callicles) [reply apps/framework-cli/src/cli/local_webserver.rs]: Hum - This method doesn't hava `Project` in it - not sure that makes the point

File: apps/framework-cli/src/cli/local_webserver.rs [hunk 3]
```diff
@@ -116,9 +138,11 @@ async fn create_client(
 struct RouteService {
     host: String,
     route_table: &'static RwLock<HashMap<PathBuf, RouteMeta>>,
+    consumption_apis: &'static RwLock<HashSet<String>>,
```
Comment (callicles) [inline apps/framework-cli/src/cli/local_webserver.rs:140]: I am not a big fan of that mutable state punched through. Not sure that there is a better solution for now though

**Review by callicles — COMMENTED** [review]
Looks like a rebase is needed - as there are some merge conflicts as well

**Review by callicles — APPROVED** [review]

**Pull request merged**